src/vue/backend/src/views/guided-setup/Confirmation.vue

```diff
@@ -0,0 +1,66 @@
+<template>
+    <idx-fullscreen customClass="gs">
+        <idx-block className="gs__hero">
+            <idx-block className="gs__icon">
+                <svg-icon icon="fal-check" />
+            </idx-block>
+            <h1>Setup Complete!</h1>
+        </idx-block>
+    </idx-fullscreen>
+</template>
+
+<script>
+import SvgIcon from '@/components/SvgIcon.vue'
+export default {
+    components: {
+        SvgIcon
+    },
+    mounted () {
+        setTimeout(() => { this.$router.push({ path: '/' }) }, 3000)
```

Comment:
If I am reading the intended behavior correctly, I believe if the user navigates away from this page, this timeout will still resolve, resulting in an unwanted path.

To solve, set the timeout to a data variable, and then on `beforeDestroy` clear it with `clearTimeout()`

```
data () {
  return () {
    timeout: ''
  }
},
mounted () {
  this.timeout = setTimeout(() => { this.$router.push({ path: '/' }) }, 3000)
},
beforeDestroy () {
  clearTimeout(this.timeout)
}